docs/source/dislib.rst

```diff
@@ -6,15 +6,18 @@ Subpackages
 
 .. toctree::
 
-    dislib.data.array.Array
+    dislib.array
```

Comment:
I am going to assume that you have checked that this works (I have had problems with links, etc. in the past)

Reply:
I checked that this works.